src/main/java/org/collection/fm/FeatureStepAnalysis.java

```diff
@@ -32,22 +27,35 @@ public static void main(String[] args) {
         if (args.length < 1) {
             System.err.println("""
                     Mandatory argument([0]): Input path
-                    Optional Argument([1]): Output path results
-                    Optional Argument([2]): Output path time
-                    Optional Argument([3]): Output path runstatus
+                    Optional Argument([1]): Analysis category: All (chosen if not set), Default, Simple, CNF, Satzilla
+                    Optional Argument([2]): Output path results
+                    Optional Argument([3]): Output path time
+                    Optional Argument([4]): Output path runstatus
                     """);
             System.exit(-1);
         }
 
-        EnumMap<AnalysisStepsEnum, Integer> enumMap = new EnumMap<>(AnalysisStepsEnum.class);
-
-        Arrays.stream(AnalysisStepsEnum.values()).forEach(e -> enumMap.put(e, 60));
+        EnumMap<AnalysisStepsEnum, Integer> enumMap = getEnumMapByCategory(args.length > 1 ? args[1] : "All", 60);
 
         analysis.initializeHandler(enumMap);
 
         List<File> files = FileUtils.getFileList(args[0]);
 
-        analysis.handleFiles(files, args.length == 1 ? "result.csv" : args[1], args.length == 1 ? "time.csv" : args[2], args.length == 1 ? "runstatus.csv" : args[3]);
+        analysis.handleFiles(files, args.length < 3 ? "result.csv" : args[1], args.length < 3 ? "time.csv" : args[2], args.length < 3 ? "runstatus.csv" : args[3]);
+    }
+
+    private static EnumMap<AnalysisStepsEnum, Integer> getEnumMapByCategory(String category, int timeout) {
```

Comment:
Do we want some kind of error handling?

Reply:
Yes , that would be good, but I think we should replace the positional args with key-value args soon, as we already have multiple optional arguments.

Reply:
A possible way could be [Apache CLI](https://commons.apache.org/proper/commons-cli/)